### ISSUE TITLE
supplychain module: Wrong current of WIOD 16 table

### ISSUE THREAD

**Issue by mastaia** [CLIMADA-project/climada_petals#113]
In the module:
https://github.com/CLIMADA-project/climada_petals/blob/develop/climada_petals/engine/supplychain.py

On line 304, the mriot unit is assigned in M.EUR. 

According to the official data download https://www.rug.nl/ggdc/valuechain/wiod/ and opening the excel table, the unit must be in M.USD
<img width="283" alt="image" src="https://github.com/CLIMADA-project/climada_petals/assets/116871277/f96ce689-775c-4bf8-b733-fb8bc03e6aa5">

**Comment by peanutfun** [comment]
Thanks for the report. @aleeciu can you confirm this and maybe come up with a fix?

**Comment by aleeciu** [comment]
Yes, that's an error - thanks for spotting. It should not really affect any other piece of the code though, it's really a results interpretation issue.

It will be fixed in https://github.com/CLIMADA-project/climada_petals/pull/81.

**Comment by mastaia** [comment]
Agree, it will not affect the results, but just the interpretation. 
Thanks a lot.
